src/cljdoc/render/articles.clj

```diff
@@ -19,6 +19,11 @@
         [:a.blue.link {:href (util/github-url :userguide/scm-faq)} "properly set SCM info"]
         " in your project."])])
 
+(defn main-list [doc-tree]
+  (when doc-tree
+    [:div.mb4.js--articles
```

Comment:
Please grep the codebase to ensure any JS code relying on this `.js--articles` class isn't negatively affected by this. I think there's only `doctree.js` which handles expanding of subtrees, which is not applicable here so it might be fine to just remove the class from this element.